extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -437,17 +437,26 @@ export class SubscriptionService {
    * @return {!Promise<boolean>}
    */
   delegateActionToLocal(action) {
-    const localPlatform = /** @type {LocalSubscriptionPlatform} */ (
-      dev().assert(this.platformStore_.getLocalPlatform(),
-          'Local platform is not registered'));
-    // TODO: add which service is passing this event
+    this.delegateActionToService(action, 'local');
+  }
+
+  /**
+   * Delegates an action to specified platform
+   * @param {string} action
+   * @param {string} serviceId
+   * @return {!Promise<boolean>}
+   */
+  delegateActionToService(action, serviceId) {
+    const platform = dev().assert(this.platformStore_.getPlatform(serviceId),
```

Comment:
This was ok as a local that's available sync. But will break regularly with other platforms loaded in separate extensions. I.e. `getPlatform('subscribe.google.com')` will fail if the extension hasn't loaded yet. We'd need to use a promisable API for this. It's ok to start with a `TODO` here and revisit in a follow up PR.

Reply:
Done